components/TARGET_PSA/services/crypto/COMPONENT_PSA_SRV_IPC/crypto_struct_ipc.h

```diff
@@ -47,9 +47,10 @@ struct psa_crypto_generator_s {
     psa_handle_t handle;
 };
 
+#define PSA_CRYPTO_GENERATOR_INIT { PSA_NULL_HANDLE }
 static inline struct psa_crypto_generator_s psa_crypto_generator_init(void)
 {
-    const struct psa_crypto_generator_s v = { 0 };
+    const struct psa_crypto_generator_s v = PSA_CRYPTO_GENERATOR_INIT;
```

Comment:
Why did you make this change? There's nothing wrong with it, but it shouldn't have been necessary: `psa_crypto_generator_t v = { 0 }` is supposed to be a valid initialization.

Reply:
Agree, the change is not necessary but also it doesn't hurt.